test/RunTests

```diff
@@ -2359,4 +2359,10 @@ train-sets/ref/slates_simple_w_interactions.stderr
 {VW} -d train-sets/cb_load.dat --cb_explore_adf -q UA --regcb -i models/regcb_ld.model -t
     train-sets/ref/regcb_load.stderr
 
+# Test 324: test extra_metrics with djson ccb
+# --quiet bc logging is not thread safe, so diff fails randomnly
+{VW} -d train-sets/ccb_data_parse_error.json --dsjson --ccb_explore_adf --extra_metrics metrics_ccb_parse_error.json --quiet
```

Comment:
(super nit, not too fussy): just so that we don't go looking for it if needed, maybe add a comment here that line 7 in data file, Timestamp field is malformed ?